Makefile

```diff
@@ -359,6 +359,18 @@ octant-antrea-ubuntu:
 	docker tag antrea/octant-antrea-ubuntu:$(DOCKER_IMG_VERSION) projects.registry.vmware.com/antrea/octant-antrea-ubuntu
 	docker tag antrea/octant-antrea-ubuntu:$(DOCKER_IMG_VERSION) projects.registry.vmware.com/antrea/octant-antrea-ubuntu:$(DOCKER_IMG_VERSION)
 
+.PHONY: antrea-mc-controller-ubuntu
+antrea-mc-controller-ubuntu:
```

Comment:
Isn't the image distroless (https://github.com/antrea-io/antrea/pull/3015/files#diff-c93bee7c19e0608e99c1726cfcd22baf2d132b73a8c567fbb5c5699cf40198c1R14)? If yes, why do we have ubuntu in the name?

Reply:
ah,yes, let me rename it. thanks.